### ISSUE TITLE
Can't launch test with new 0.1.5 version

### ISSUE THREAD

**Issue by ianrenauld** [itszero/wdio-browserstack-service#7]
I randomly tried a new install of my test framework and much to my surprise I could not launch test on BrowserStack.  After some digging, the only difference between my normal install and the new one is a bump to version 0.1.5 of wdio-browserstack-service. I bumped up to that version on my normal install and it started failing as well. 

Running on a MacBook Pro on Sierra, Webdriver.io version 4.8.1.

Here is what I get after launching my test on BrowserStack:
```
/Users/ianrenauld/dev/wdioTest/node_modules/webdriverio/build/lib/cli.js:429
            throw e;
            ^

Error: Couldn't initialise launcher from service "browserstack".
Error: Cannot find module './build/browserstack-launch-service'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/ianrenauld/dev/wdioTest/node_modules/wdio-browserstack-service/launcher.js:1:99)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Launcher.getLauncher (/Users/ianrenauld/dev/wdioTest/node_modules/webdriverio/build/lib/launcher.js:745:35)
    at Launcher._callee$ (/Users/ianrenauld/dev/wdioTest/node_modules/webdriverio/build/lib/launcher.js:229:49)
    at tryCatch (/Users/ianrenauld/dev/wdioTest/node_modules/regenerator-runtime/runtime.js:64:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (/Users/ianrenauld/dev/wdioTest/node_modules/regenerator-runtime/runtime.js:299:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (/Users/ianrenauld/dev/wdioTest/node_modules/regenerator-runtime/runtime.js:116:21)
    at step (/Users/ianrenauld/dev/wdioTest/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at /Users/ianrenauld/dev/wdioTest/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
    at Promise.F (/Users/ianrenauld/dev/wdioTest/node_modules/core-js/library/modules/_export.js:35:28)
    at Launcher.<anonymous> (/Users/ianrenauld/dev/wdioTest/node_modules/babel-runtime/helpers/asyncToGenerator.js:14:12)
    at Launcher.run (/Users/ianrenauld/dev/wdioTest/node_modules/webdriverio/build/lib/launcher.js:375:29)
```

**Comment by pgAdmin** [comment]
I am facing it too. For now reverted to 0.1.4

**Comment by itszero** [comment]
Yeah. Sorry about that. I forgot to build it before release for 0.1.5. I’ve unpublished it so no one will fall for it again and I’ve published 0.1.7 to fix this.

Thank you very much for the instant feedbacks!
